laravel/the-basics/customizing-deployments.html.md

```diff
@@ -0,0 +1,35 @@
+---
+title: Customizing Deployments
+layout: framework_docs
+objective: Customize how Fly deploys your Laravel application.
+order: 6
+---
+
+The `fly launch` command sets you up with a useful starting point, but you can customize just about everything to meet your needs. Here's a few things that might be useful to know.
+
+## User Scripts
+
+The `fly launch` command generated a `.fly/scripts` directory. Any file in here ending in `.sh` will run (via `bash`) anytime your application is started.
+
+User scripts are run from the `ENTRYPOINT` script before anything else is ran.
+
+By default, `fly launch` generates the script `.fly/scripts/caches.sh`, which runs various artisan cache commands such as `config:cache`, `route:cache`, and `view:cache`.
+
+You can enable or disable those as you see fit, as well as add your own `.sh` scripts!
+
+## Release Command
+
+You may want to use the [`release_command`](/docs/reference/configuration/#the-deploy-section) to perform database migrations or other tasks. 
+
+These types of tasks could be a `.sh` user script, or `release_command` defined in your `fly.toml` file - it's up to you and your use case.
```

Comment:
@fideloper  In here, by ".sh user script", you mean that it has to be in .fly/scripts right?

Reply:
If the location is a requirement, maybe you could add "...user script in .fly/scripts..." in this line

Reply:
or reference the ## User Scripts section with a link [user script](/docs/laravel/the-basics/customizing-deployments/#user-scripts)